src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -109,44 +111,35 @@ def __init__(
         method: str = "predict",
         estimated_coef=None,
         sigma_X=None,
-        params_lasso_screening={
-            "alpha": None,
-            "n_alphas": 10,
-            "alphas": None,
-            "alpha_max_fraction": 0.5,
-            "cv": 5,
-            "tol": 1e-6,
-            "max_iter": 1000,
-            "fit_intercept": False,
-            "selection": "cyclic",
-        },
-        params_lasso_distillation_x=None,
+        lasso_screening=LassoCV(n_alphas=10, cv=5, tol=1e-6, fit_intercept=False),
+        model_distillation_x=LassoCV(n_jobs=1, n_alphas=10),
         refit=False,
-        screening=True,
         screening_threshold=10,
         centered=True,
         n_jobs=1,
         joblib_verbose=0,
         fit_y=False,
         scaled_statistics=False,
-        random_state=2022,
+        random_state=None,
+        reuse_screening_model=True,
     ):
         self.estimator = estimator
         _check_vim_predict_method(method)
         self.estimated_coef = estimated_coef
         self.method = method
         self.sigma_X = sigma_X
-        self.params_lasso_screening = params_lasso_screening
-        self.params_lasso_distillation_x = params_lasso_distillation_x
+        self.lasso_screening = lasso_screening
+        self.model_distillation_x = model_distillation_x
         self.refit = refit
-        self.screening = screening
         self.screening_threshold = screening_threshold
         self.centered = centered
         self.n_jobs = n_jobs
         self.joblib_verbose = joblib_verbose
         self.fit_y = fit_y
         self.scaled_statistics = scaled_statistics
         self.random_state = random_state
+        self.is_lasso = isinstance(estimator, (Lasso, LassoCV))
```

Comment:
Why do you need this attribute?
It seems only used once.

Reply:
If you want to keep it, you should add an underline to this end and put it after the attributes defined by the parameters.

Reply:
I will remove it, it will not be used in the future after taking care of #383